### DISCUSSION TITLE
Using Borg beta 2.0.0.bx with Vorta 0.10.0-beta1

### DISCUSSION THREAD

**Discussion by aliagowani** [borgbase/vorta#2111] [FAQ]
I have searched all over the place and am not sure how to do this. I've read that Vorta should be able to use call Borg 2.x beta releases. Is that correct? 

Vorta for me is using Borg located under /Applications/Vorta.app/Contents/Resources/borg-dir/borg.exe  This version of Borg is 1.4.0. From my reading and understanding, I need to compile a Windows executable from Borg 2.x beta and then replace it?

If this is correct, how do I compile Windows executable file? For what I read, I need to execute the compiler in Python in a Windows environment and call the borg script? Can someone provide further instruction on how to do this? 

Thank you for helping and your input.

**Comment by m3nu** [comment]
From what I know there is no Windows executable of Borg right now. There used to be a few years ago, but the person publishing them stopped at some point. There should be an open issue in the Borg repo on the current state.

**Comment by aliagowani** [comment]
I was able to find the windows executable (it's in the Macos package, link below) and replace it in the Vorta app (under resources borg-dir folder). However I encounter an error in Vorta when clicking "Start Backup", even when a new repository has been created. Many new variables/parameters are listed in the error. 

FYI: when I go to Settings / About in Vorta, it does show the Borg beta version so it seems to read and recognize it.

borg-macos1012.tgz https://github.com/borgbackup/borg/releases/download/2.0.0b11/borg-macos1012.tgz

**Reply by goebbe** [reply]
Since the start of September, there have been quite some fundamental changes to the latest versions of borg beta: 
https://borgbackup.readthedocs.io/en/master/changes.html
@aliagowani So borg beta (on the command line) works? But Vorta using borg beta (which version) does lead to errors?

**Reply by aliagowani** [reply]
I did not try commandline. Just replaced the file in the package and tried to run it.
